kt/plugins/godot-intellij-plugin/src/IJ183/kotlin/godot/intellij/plugin/extension/moduleExt.kt

```diff
@@ -0,0 +1,6 @@
+package godot.intellij.plugin.extension
+
+import com.intellij.openapi.module.Module
+import org.jetbrains.kotlin.idea.framework.isGradleModule
+
+fun Module.isGradleModule(): Boolean = this.isGradleModule()
```

Comment:
Why is this necessary ?

Reply:
Because jetbrains moved the function `isGradleModule` to another package.
In older sdk versions, it's now at the old path and in newer ones it's at the new path, breaking compatibility.
This file is a wrapper which links to the correct path.
A different version is included now in our plugin based on the target version of IDEA we're building for.